08_files_lab/02.py

```diff
@@ -1,7 +1,20 @@
-"""
-Write a program that takes 3 names. The first two
-are existing file names and the last is a new file name
-Program should write the lines from the first two 
-files interwinded into the output file
-"""
-
+'''
+Usage: 02.py <firstFile> <secondFile> <targetFile>
+Alternately writes lines from firstFile and secondFile to targetFile
+'''
+
+import sys
+import os
+import itertools
+
+if len(sys.argv) != 4 or not os.path.exists(sys.argv[1]) or not os.path.exists(sys.argv[2]):
+    print 'Usage: %s <firstFile> <secondFile> <targetFile>' % sys.argv[0]
+    sys.exit(1)
+
+with open(sys.argv[1], 'r') as first_file:
+    with open(sys.argv[2], 'r') as second_file:
+        with open(sys.argv[3], 'w') as output_file:
+            for line in itertools.izip_longest(first_file, second_file, fillvalue=''):
+                for subline in line:

```

Comment:
Better to split on the for line:
`for line1, line2 in itertools.izip_longest(...)`